### PR TITLE
Fix #1084: quick-api-reference sample

### LINKED ISSUE #1084: The Quick Start tutorial "quick-api-reference" sample doesn't work
**Describe the bug**
quick-api-reference sample tip function doesn't work
I guess it because the [API reference](https://developer.chrome.com/docs/extensions/reference/) is upgrade

**To Reproduce**
install the quick-api-reference extension，tip function doesn't work。

### PR DESCRIPTION
**Pull request by Pask00**
Fix #1084

### CODE REVIEW

File: functional-samples/tutorial.quick-api-reference/content.js [hunk 1]
```diff
@@ -27,4 +22,4 @@
 function createDomElement(html) {
   const dom = new DOMParser().parseFromString(html, 'text/html');
   return dom.body.firstElementChild;
-}
+}
```
Comment (oliverdunk) [inline functional-samples/tutorial.quick-api-reference/content.js:25]: Could we keep this newline?

File: functional-samples/tutorial.quick-api-reference/content.js [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
   const { tip } = await chrome.runtime.sendMessage({ greeting: 'tip' });
 
   const tipWidget = createDomElement(`
-    <button class="navigation-rail__link" popovertarget="tip-popover" popovertargetaction="show" style="padding: 0; border: none; background: none;>
-      <div class="navigation-rail__icon">
-        <svg class="icon" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" width="24" height="24" viewBox="0 0 24 24" fill="none"> 
-        <path d='M15 16H9M14.5 9C14.5 7.61929 13.3807 6.5 12 6.5M6 9C6 11.2208 7.2066 13.1599 9 14.1973V18.5C9 19.8807 10.1193 21 11.5 21H12.5C13.8807 21 15 19.8807 15 18.5V14.1973C16.7934 13.1599 18 11.2208 18 9C18 5.68629 15.3137 3 12 3C8.68629 3 6 5.68629 6 9Z'"></path>
-        </svg>
-      </div>
+    <button type="button" popovertarget="tip-popover" popovertargetaction="show" style="padding: 0;">
```
Comment (oliverdunk) [inline functional-samples/tutorial.quick-api-reference/content.js:9]: Removing the icon seems reasonable (it didn't look great anymore) but now the button is quite awkwardly sized compared to everything else. Could you have a play with some simple tweaks that might help with that?

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Thanks for the contribution! I've left a couple of suggestions - please also look at signing the CLA which is needed before we can accept this.

File: functional-samples/tutorial.quick-api-reference/content.js (hunk 1 above)
Comment (Pask00) [reply functional-samples/tutorial.quick-api-reference/content.js]: Yes, sorry!

File: functional-samples/tutorial.quick-api-reference/content.js (hunk 2 above)
Comment (Pask00) [reply functional-samples/tutorial.quick-api-reference/content.js]: Added some changes looking at the "Sign In" button in the same navigation bar: so set a specific height, font and added some padding

Comment (oliverdunk) [reply functional-samples/tutorial.quick-api-reference/content.js]: Much better, thanks!

File: functional-samples/tutorial.quick-api-reference/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Overview
+
+The extension built allows users to quickly navigate to Chrome API reference pages using the omnibox.
+
+The events from the following APIs have been handled: `chrome.runtime`, `chrome.omnibox` and `chrome.alarms`.
+
+The complete tutorial is available [here](https://developer.chrome.com/docs/extensions/get-started/tutorial/service-worker-events).
+
+**Note:** the GIFs in the tutorial are showing a previous version of [Chrome for Developers](https://developer.chrome.com).
```
Comment (oliverdunk) [inline functional-samples/tutorial.quick-api-reference/README.md:13]: I've just updated our documentation, so going to remove this.

```suggestion
```

**Review by oliverdunk — APPROVED** [review]
Thanks again! This looks good so going to merge a couple of tweaks and then land this. I've updated the tutorial to reflect the new code and appearance of the site.

File: functional-samples/tutorial.quick-api-reference/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+This sample demonstrates how to handle events with service workers.
+
+## Overview
+
+The extension built allows users to quickly navigate to Chrome API reference pages using the omnibox.
+
+The events from the following APIs have been handled: `chrome.runtime`, `chrome.omnibox` and `chrome.alarms`.
+
+The complete tutorial is available [here](https://developer.chrome.com/docs/extensions/get-started/tutorial/service-worker-events).
+
```
Comment (oliverdunk) [inline functional-samples/tutorial.quick-api-reference/README.md:12]:
```suggestion
```

**Pull request merged**